docs/CHANGELOG/changelog-0.7.x.md

```diff
@@ -0,0 +1,53 @@
+# Claudie `v0.7`
+
+!!! warning "Due to using the latest version of longhorn the `v0.7.x` will not be backwards compatible with `v0.6.x`"
+
+## Deployment
+
+To deploy Claudie `v0.7.X`, please:
+
+1. Download the archive and checksums from the [release page](https://github.com/berops/claudie/releases)
+
+2. Verify the archive with the `sha256` (optional)
+
+    ```sh
+    sha256sum -c --ignore-missing checksums.txt
+    ```
+
+   If valid, output is, depending on the archive downloaded
+
+    ```sh
+    claudie.tar.gz: OK
+    ```
+
+   or
+
+    ```sh
+    claudie.zip: OK
+    ```
+
+   or both.
+
+3. Lastly, unpack the archive and deploy using `kubectl`
+
+   > We strongly recommend changing the default credentials for MongoDB, MinIO and DynamoDB before you deploy it. To do this, change contents of the files in `mongo/secrets`, `minio/secrets` and `dynamo/secrets` respectively.
+
+    ```sh
+    kubectl apply -k .
```

Comment:
Why do we recommend downloading the source code in tar format, when there is one big yaml file containing everything needed?

Reply:
added both ways

Reply:
Also, we don't have checksums for the ZIP and TAR files in our checksum TXT. At least, I haven't found them in the latest release. Did something change?

Reply:
I can see them there

Reply:
https://github.com/berops/claudie/releases/download/v0.6.6/claudie_checksum.txt

Reply:
![image](https://github.com/berops/claudie/assets/43497822/db765ee7-0787-450c-a020-36b1d0b5b5c6)

Reply:
fixed, now we only mention the yaml files